packages/kbn-guided-onboarding/src/components/landing_page/guide_card.test.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { GuideCard } from './guide_card';
+
+jest.mock('../../kibana_services', () => {
+  const { uiSettingsServiceMock, httpServiceMock } = jest.requireActual('@kbn/core/public/mocks');
```

Comment:
These can be imported directly from their packages:
`uiSettingsServiceMock` from `@kbn/core-ui-settings-browser-mocks`
`httpServiceMock` from `@kbn/core-http-browser-mocks`